File: fibonacci.py
def fibonacci(n):
    if n < 2:
        return n
    firstNum = 0
    secondNum = 1
    fibNum = 0
    counter = 1
    while n > counter:
        fibNum = firstNum + secondNum
        firstNum = secondNum
        secondNum = fibNum
        counter += 1
        # print(counter)
    return fibNum

File: fibonacci.js
const fibonacci = (num) => {
    if (num < 2){
        return num
    }
    let counter = 1;
    let firstNum = 0;
    let secondNum = 1;
    let fibNum;
    while (num > counter){
        fibNum = firstNum + secondNum
        firstNum = secondNum
        secondNum = fibNum
        // console.log(fibNum)
        counter ++
    }
    return fibNum
}

module.exports = {fibonacci}